Makefile

```diff
@@ -4,12 +4,16 @@ isort = isort -rc src tests
 black = black src tests
 
 VERSION := $(shell python3 setup.py -V)
-RPMDIR := "$(shell pwd)/rpms-el7"
+RPMDIR-EL7 := "$(shell pwd)/rpms-el7"
 
 ifndef $(RELEASE)
 RELEASE := dev
 endif
 
+ifndef $(ISO_MAJOR_VERSION)
+ISO_MAJOR_VERSION := 3
```

Comment:
Is this safe? Shouldn't we pick the general availibility ISO here to make sure we don't accidentally publish minor version increments to ISO3?